index.js

```diff
@@ -32,6 +34,19 @@ module.exports = {
   paths: paths,
   absolutePaths: absolutePaths,
   treeForVendor: function() {
+
+    var jqueryPath;
+    try {
+      jqueryPath = path.dirname(resolve.sync('jquery/package.json', { basedir: this.project.root }));
+    } catch (error) {
+      jqueryPath = path.dirname(require.resolve('jquery/package.json'));
+    }
```

Comment:
If we don't have a `jqueryPath` at this point something else is going to have resulted in this breaking _long_ before we got this far. The one in the `catch` statement is depended upon by `ember-source` itself.